tests/page-objects/wallet.selectors.ts

```diff
@@ -1,3 +1,4 @@
 export enum WalletPageSelectors {
   BtnSendTokens = 'btn-send-tokens',
+  statusMessage = 'status-message',
```

Comment:
This hasn't been changed yet. If you didn't pull the changes we did yesterday, you may have force pushed over them

Reply:
No I pulled it all down but changed that breaks my test so I changed it back. You guys can refactor how you want once this is pushed into dev. But I want to merge as is.

Reply:
When making one of these Github code suggestions, it's a suggestion, not a 100% promise it'll work. It may require additional changes on your part. In this case, as the name has been updated here, it may need updating elsewhere in your code.

In all of our tests, the selectors follow a certain naming convention pascal case (starting with a capital letter). It'd be great if we could conform to the existing standards. Happy to help you track down where the issue lies.